Network Class/Network.h

```diff
@@ -71,6 +72,13 @@ class Network
 	bool				compareOutput(const Matrix&);
 	const Matrix		hadamardProduct(const Matrix&, const Matrix&);
 	layerReport			outputLayerReport();
+	//---------------------Methods to check the typo errors by the users--------------------
+	void				checkLearningRate(int = 1);		//the highest value for the learning rate is set to 1 but can be changed while calling the function
```

Comment:
Setter/mutator methods contain the checks so they can be `public`!

Reply:
oh yes! You're right!